docs/zh/connector-v2/sink/Paimon.md

```diff
@@ -0,0 +1,87 @@
+# Paimon
+
+> Paimon 数据连接器
+
+## 描述
+
+Apache Paimon数据连接器。支持cdc写以及自动建表。
+
+## 主要特性
+
+- [x] [exactly-once](../../concept/connector-v2-features.md)
+
+## 连接器选项
+
```

Comment:
There seem to be some problems with the format

Reply:
> There seem to be some problems with the format

I've already previewed it and there don't seem to be any formatting issues here.